core/templates/dev/head/pages/exploration_editor/exploration_title_directive.html

```diff
@@ -0,0 +1,10 @@
+<p ng-class="{'has-error': !explorationTitleService.displayed.length > 0}" ng-style="<[::pStyle]>">
+  <label ng-style="<[::formStyle]>" ><[::title]></label>
+  <input id="explorationTitle" type="text" class="form-control protractor-test-exploration-title-input" ng-model="explorationTitleService.displayed" ng-blur="saveExplorationTitle()" placeholder="Choose a title for your exploration." focus-on="<[::EXPLORATION_TITLE_INPUT_FOCUS_LABEL]>" maxlength="40" ng-trim="false">
```

Comment:
Break this up a bit -- e.g. start ng-model on a new line and left-align it with the id attr in the line above it. Easier to read.